### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by NemTechCN**

### CODE REVIEW

File: src/language/en-US.ts [hunk 1]
```diff
@@ -138,7 +138,7 @@ export const en_US = {
     collection_record: 'Transfers received',
     enter_asset_type_alias_or_address_search: 'Enter transaction hash search',
     ordinary_transfer: 'Transfer',
-    Multisig_transfer: 'Multisig transfer',
+    Multi_sig_transfer: 'Multi-sig transfer',
```
Comment (decentraliser) [inline src/language/en-US.ts]: multisig is good

we have to write technical  like in the documentation https://nemtech.github.io/concepts/multisig-account.html

File: src/language/en-US.ts [hunk 2]
```diff
@@ -406,7 +407,7 @@ export const en_US = {
     keep_it_in_a_safe_place_on_the_isolated_network_private_key: 'Keep it in a safe place on the isolated network. Do not share and store private key in a networked environment, such as emails, photo albums, social applications, etc.',
     please_accurately_copy_the_secure_backup_private_key: 'Please accurately copy the secure backup private key',
     display_private_key_QR_code: 'Display QR code',
-    display_private_key: 'Privatekey',
+    display_private_key: 'PrivateKey',
```
Comment (decentraliser) [inline src/language/en-US.ts]: private key

File: src/language/en-US.ts [hunk 3]
```diff
@@ -462,7 +463,7 @@ export const en_US = {
     Public_wallet: 'Public wallet',
     Wallet_name: 'Name',
     Wallet_address: 'Address',
-    publickey: 'Publickey',
+    publickey: 'PublicKey',
```
Comment (decentraliser) [inline src/language/en-US.ts]: Public key

Comment (NemTechCN) [reply src/language/en-US.ts]: fixed

File: src/language/en-US.ts (hunk 2 above)
Comment (NemTechCN) [reply src/language/en-US.ts]: fixed

**Pull request merged**